app/v2/notifications/post_notifications.py

```diff
@@ -135,6 +140,24 @@ def post_precompiled_letter_notification():
     return jsonify(resp), 201
 
 
+def _seed_bounce_data(epoch_timestamp: int, service_id: str):
+    current_time_ms = int(1000.0 * datetime.now().timestamp())
+    time_difference_ms = current_time_ms - epoch_timestamp
+
+    twenty_four_hours_ms = 86400000
```

Comment:
Let's write the math for this so it's easier to see where it comes from
```suggestion
    TWENTY_FOUR_HOURS_MS = 24 * 60 * 60 * 1000
```